app/models/agents/pushbullet_agent.rb

```diff
@@ -71,7 +65,8 @@ def validate_api_key
     end
 
     def complete_device_id
-      devices.map { |d| {text: d['nickname'], id: d['iden']} }
+      devList = devices.map { |d| {text: d['nickname'], id: d['iden']} }
+      devList.unshift({:text=>"All Devices", :id=>"__ALL__"})
```

Comment:
Ruby let's you chain function calls, with that you do not need the `devList` variable:
```
devices
  .map { |d| {text: d['nickname'], id: d['iden']} }
  .unshift(text: 'All Devices', id: '__ALL__')
``` 
an other option is to have it all in one line
```
devices.map { |d| {text: d['nickname'], id: d['iden']} }.unshift(text: 'All Devices', id: '__ALL__')
```
We are not very strict about max line length, so it's up to you which version you like better. However we prefer the new `key: 'value'` hash style over the old hashrocket (`=>`)